apps/static/dance/ai/model/cached-spacy-background-map.json

```diff
@@ -0,0 +1,208 @@
+{
+  "emojiAssociations": {
+    "poopy": {

```

Comment:
Non-blocking comment for v2 (probably best to answer after merging). Can we use the emojis directly rather than our interpretation of the emojis?

Reply:
See follow-up discussion on this point within this slack thread: https://codedotorg.slack.com/archives/C05P12U42F5/p1697067515602939